packages/usa-in-page-navigation/src/styles/_usa-in-page-navigation.scss

```diff
@@ -0,0 +1,121 @@
+@use "uswds-core" as *;
+@use "sass:color";
+
+.usa-in-page-nav-container {
+  align-items: flex-start;
+  display: flex;
+
+  // Set main landmark max width to ensure in-page nav will fit
+  main,
+  [class^="main"] {
+    max-width: units($theme-in-page-nav-main-content-max-width);
+    width: 100%;
+  }
+
+  .usa-in-page-nav__heading {
```

Comment:
Nitpick, but I always like to order my selectors based on how they're ordered in markup. In this case `usa-in-page-nav ` would be before heading.

```
.usa-in-page-nav-container
  .usa-in-page-nav
    nav
      .usa-in-page-nav__heading
```

Reply:
I've also noticed everything is nested inside of `usa-in-page-nav-container`. Was this intentional?

Reply:
Yes there is a requirement detailed in the [Using the in-page navigation component](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/mm-inclusive-patterns-in-page-navigation/components/in-page-navigation/) notes that calls for the addition of a wrapper with the class usa-in-page-nav-container.

Reply:
But does the CSS need this specificity? We may be able to remove some unnecessary specificity by flattening the code

Reply:
I will try flattening and reordering